### PR TITLE
chore: fix whitespace format issues

### PR DESCRIPTION
**Pull request by doitian**

### CODE REVIEW

**Comment by doitian** [comment]
bors r=quake,u2,keroro520,zhangsoledad

**Comment by doitian** [comment]
bors retry

**Comment by doitian** [comment]
bors retry

**Comment by doitian** [comment]
bors r=quake,u2,keroro520,zhangsoledad

**Comment by doitian** [comment]
bors r-

```

0.04s
$ make check-whitespaces
218
git diff-index --check --cached $(git rev-parse --verify master 2>/dev/null || echo "4b825dc642cb6eb9a060e54bf8d69288fbee4904") --
219
protocol/src/protocol_generated.rs:4118: new blank line at EOF.
220
rpc/README.md:1512: new blank line at EOF.
221
script/data-loader/Cargo.toml:11: new blank line at EOF.
222
util/multisig/Cargo.toml:15: new blank line at EOF.
223
util/network-alert/Cargo.toml:29: new blank line at EOF.
224
util/network-alert/src/alert_signature.toml:11: new blank line at EOF.
225
Makefile:103: recipe for target 'check-whitespaces' failed
226
make: *** [check-whitespaces] Error 2
227
The command "make check-whitespaces" exited with 2.
```

File: Makefile [hunk 1]
```diff
@@ -93,12 +93,15 @@ bench-test:
...
-ci: fmt clippy test bench-test check-cargotoml check-dirty-rpc-doc security-audit
+ci: fmt clippy test bench-test check-cargotoml check-whitespaces check-dirty-rpc-doc security-audit
 	git diff --exit-code Cargo.lock
 
 check-cargotoml:
 	./devtools/ci/check-cargotoml.sh
 
+check-whitespaces:
+	git diff-index --check --cached $$(git rev-parse --verify master 2>/dev/null || echo "4b825dc642cb6eb9a060e54bf8d69288fbee4904") --
+
```
Comment (u2) [inline Makefile]: what's this `4b825dc642cb6eb9a060e54bf8d69288fbee4904`?

File: protocol/src/protocol_generated.rs [hunk 2]
```diff
@@ -4115,4 +4115,3 @@ pub fn finish_size_prefixed_sync_message_buffer<'a, 'b>(fbb: &'b mut flatbuffers
 }
 }  // pub mod Protocol
 }  // pub mod Ckb
-
```
Comment (u2) [inline protocol/src/protocol_generated.rs]: this file is generated.

File: Makefile (hunk 1 above)
Comment (doitian) [reply Makefile]: Git genesis commit

File: protocol/src/protocol_generated.rs (hunk 2 above)
Comment (doitian) [reply protocol/src/protocol_generated.rs]: I think it is too strick to check the blank new lines at the end of the file.

Comment (doitian) [reply protocol/src/protocol_generated.rs]: Just disable the eof blank lines checker.

**Comment by u2** [comment]
bors r=quake,u2,keroro520,zhangsoledad

**Pull request merged**